projects/onnx/build.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# Copyright 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+set -e
+
+cp -r /onnx /onnx_${TARGET}
```

Comment:
It is possible to build everything in same directory, just apply a proper cleanup instead of `python3 setup.py clean` (clean command not implemented in onnx):
```
rm -rf build /onnx/.setuptools-cmake-build *.o
```